### PR TITLE
Implement "top 11 + bac validator" rule in miner and validator

### PR DESCRIPTION
**Pull request by emnoor-reef**

### CODE REVIEW

File: compute_horde/compute_horde/utils.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import bittensor
+from more_itertools import one
+
+if TYPE_CHECKING:
+    from bittensor.chain_data import NeuronInfo
+
+logger = logging.getLogger(__name__)
+
+BAC_VALIDATOR_SS58_ADDRESS = '5HBVrFGy6oYhhh71m9fFGYD7zbKyAeHnWN8i8s9fJTBMCtEE'
+VALIDATORS_LIMIT = 12
```
Comment (agoncharov-reef) [inline compute_horde/compute_horde/utils.py:13]: Proposal: move all settings (even hardcoded ones) to `settings.py`

File: validator/app/src/compute_horde_validator/validator/tasks.py [hunk 2]
```diff
@@ -50,13 +51,13 @@ def _run_synthetic_jobs():
 @app.task()
 def run_synthetic_jobs():
     if not settings.DEBUG_DONT_STAGGER_VALIDATORS:
-        metagraph = bittensor.metagraph(settings.BITTENSOR_NETUID, settings.BITTENSOR_NETWORK)
+        validators = get_validators(netuid=settings.BITTENSOR_NETUID, network=settings.BITTENSOR_NETWORK)
         my_key = settings.BITTENSOR_WALLET().get_hotkey().ss58_address
-        validator_keys = sorted([n.hotkey for n in metagraph.neurons if
+        validator_keys = sorted([n.hotkey for n in validators if
                                  n.validator_permit and n.stake.tao >= MEANINGFUL_VALIDATOR_STAKE_THRESHOLD_TAO])
```
Comment (agoncharov-reef) [inline validator/app/src/compute_horde_validator/validator/tasks.py:57]: `.validator_permit` shouldn't be here, right?

File: compute_horde/compute_horde/utils.py (hunk 1 above)
Comment (emnoor-reef) [reply compute_horde/compute_horde/utils.py:13]: This is the library, no `settings.py` here.

Comment (agoncharov-reef) [reply compute_horde/compute_horde/utils.py:13]: Could've been. Ok

**Pull request merged**